### PR TITLE
Enhancement: Make Migrations optional

### PR DESCRIPTION
**Pull request by CruzMolina**
- try/catch require's of `Migrations.sol` in
`@truffle/migrate` to ignore non-existence of
Migrations artifact

### CODE REVIEW

File: scripts/ci.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ elif [ "$QUORUM" = true ]; then
   sudo mv docker-compose /usr/local/bin
   git clone https://github.com/jpmorganchase/quorum-examples
   cd quorum-examples
-  QUORUM_GETH_ARGS="-allow-insecure-unlock" docker-compose up -d
+  docker-compose up -d
```
Comment (seesemichaelj) [inline scripts/ci.sh:52]: how is this change related?

Comment (seesemichaelj) [reply scripts/ci.sh:52]: I'm happy with the rest of the code, so I defer to @gnidan's review. Seems like this change was added because Quorum changed somethings in `quorum-examples`, causing tests to fail (though I see that they're passing). I'd rather have this fixed in another PR and have this one rebased after the merge of that PR

Comment (gnidan) [reply scripts/ci.sh:52]: I agree with @seesemichaelj here. Better to make separate PRs out of unrelated changes

**Comment by lionelyoung** [comment]
Does this mean we can track state ourselves and not use migrations?

**Comment by gnidan** [comment]
Yep, totally optional!
